converters/fromZigbee.js

```diff
@@ -8575,6 +8576,74 @@ const converters = {
             return result;
         },
     },
+    PC321_metering: {
```

Comment:
Please move this converter to `fzLocal` in `owon.js`, I assume after that this PR can be merged? (e.g. https://github.com/Koenkk/zigbee2mqtt.io/blob/94a4668735bb090b0029f4ee734297a1c37a4008/docs/externalConvertersExample/freepad_ext.js#L20 )